functions/src/index.ts

```diff
@@ -12,12 +12,14 @@
 import admin from 'firebase-admin'
 import { FieldValue, type Transaction } from 'firebase-admin/firestore'
 import { type BlockingFunction } from 'firebase-functions'
+import { onRequest } from 'firebase-functions/v1/https'
```

Comment:
We should aim to use v2 functions here.

Reply:
will be changed to an onCall anyways in the next iteration.

Reply:
Would be great of we can document it in an issue if we don't fix it here.